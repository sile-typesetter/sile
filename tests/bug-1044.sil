\begin[papersize=a6]{document}
\nofolios
\neverindent
\font[family=Libertinus Serif]{Ta of}
% If complex shaping is skipped and glyphAdvance ignored when outputting
% a series of hboxes, the "Ta" kerning above will cause "of" to be positioned
% too far forward.
\end{document}
