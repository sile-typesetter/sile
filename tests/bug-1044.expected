Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	27.9564
Set font 	Libertinus Serif;10;400;;normal;;LTR
Mx 	5.4800
Mx 	4.5700
T	53 a=5.9700 66 a=4.5700	(Ta)
Mx 	27.9627
T	80 71 w=8.1400	(of)
End page
Finish
